src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ChatEntry from './ChatEntry.js';
+import PropTypes from 'prop-types';
```

Comment:
Small nit, in JS, imports are organized first by:
1. External libraries, then 
2. Internal imports

So I would swap L2 and L3 😊